client/src/components/Card.jsx

```diff
@@ -4,26 +4,10 @@ import StarReview from './StarReview.jsx';
 import axios from 'axios';
 
 const Card = ({ product, cardType }) => {
-  const [rating, setRating] = useState(0);
```

Comment:
I see you're pulling out your API calls and using the props from the mock data. We'll need to coordinate our ultimate decision on how much API call/state work to keep at the top level vs. the 4 sections stateful components.